docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -1,22 +1,23 @@
-### Writing Pester Tests
+# Writing Pester Tests
 
-Note that this document does not replace the documents found in the [Pester](https://github.com/pester/pester "Pester") project.
+Note that this document does not replace the documents found in the [Pester](https://github.com/pester/pester) project.
 This is just some quick tips and suggestions for creating Pester tests for this project.
 The Pester community is vibrant and active, if you have questions about Pester or creating tests, the [Pester Wiki](https://github.com/pester/pester/wiki) has a lot of great information.
 As of January 2018, PowerShell Core is using Pester version 4 which has some changes from earlier versions.
 See [Migrating from Pester 3 to Pester 4](https://github.com/pester/Pester/wiki/Migrating-from-Pester-3-to-Pester-4) for more information.
 
 When creating tests, keep the following in mind:
-* Tests should not be overly complicated and test too many things
-    * boil down your tests to their essence, test only what you need
-* Tests should be as simple as they can
-* Tests should generally not rely on any other test
 
-Examples:
+- Tests should not be overly complicated and test too many things.
+    - Boil down your tests to their essence, test only what you need.
+- Tests should be as simple as they can.
+- Tests should generally not rely on any other test.
+
+## Examples
 
 Here's the simplest of tests:
 
-```powershell
+```PowerShell
```

Comment:
@ThreeFive-O these should actually be lowercase because otherwise it breaks markdownlint, please revert.

Reply:
Fixed all of them. Thank you very much @RDIL!